api/types/kustomization.go

```diff
@@ -172,6 +175,33 @@ type Kustomization struct {
 	BuildMetadata []string `json:"buildMetadata,omitempty" yaml:"buildMetadata,omitempty"`
 }
 
+const (
+	deprecatedWarningToRunEditFix              = "# Warning: Run 'kustomize edit fix' to update your Kustomization automatically."
+	deprecatedWarningToRunEditFixExperimential = "# Warning: [EXPERIMENTAL]Run 'kustomize edit fix' to update your Kustomization automatically."
+	deprecatedBaseWarningMessage               = "# Warning: 'bases' is deprecated. Please use 'resources' instead." + "\n" + deprecatedWarningToRunEditFix
```

Comment:
since this is a single problem, I would prefer to have it printed on one line, without the second "warning" text

Reply:
I restore it to one-line warning messages.